src/App.js

```diff
@@ -1,37 +1,67 @@
 import './styles/App.css';
 import TaskList from './TaskList';
 import Header from './Header';
+import Lists from './Lists';
 import Modal from './Modal.js';
-import {useState} from 'react';
+import {useEffect, useState} from 'react';
 
 function App(props) {
     const [showCompleted, setShowCompleted] = useState(true);
     const [showDeleteModal, setShowDeleteModal] = useState(false);
     const [showDelCompletedModal, setShowDelCompletedModal] = useState(false);
+    const [showDelListModal, setShowDelListModal] = useState(false);
     const [deleteID, setDeleteID] = useState(null);
+    const [modalDisplayed, setModalDisplayed] = useState(showDeleteModal || showDelCompletedModal || showDelListModal);
+
+    useEffect(() => {
+        setModalDisplayed(showDeleteModal || showDelCompletedModal || showDelListModal);
+        console.log("modal displayed");
+    },[showDeleteModal, showDelCompletedModal, showDelListModal])
 
     function toggleShowCompleted(item) {
         setShowCompleted(!showCompleted)
         console.log("Toggled showing Completed vs Uncompleted")
     }
 
+    function getWindowDimensions() {
```

Comment:
Better to use useWindowSize hook, because that'll cause the component to re-render if the window size
changes.